### PR TITLE
Supported locales and reload fix

### PR DESCRIPTION
**Pull request by ilteoood**
To solve the fix exposed in #86

### CODE REVIEW

**Comment by defint** [comment]
Language switching doesn't work.

File: lib/loaders/file_translation_loader.dart [hunk 1]
```diff
@@ -22,7 +22,7 @@ class FileTranslationLoader extends TranslationLoader implements IFileContent {
   Locale _locale;
 
   @override
-  Locale get locale => _locale ?? forcedLocale;
+  Locale get locale => forcedLocale ?? _locale;
```
Comment (defint) [inline lib/loaders/file_translation_loader.dart:25]: This line avoid all possibilities to change language.
After fixing - please add test for this case.

Comment (ilteoood) [reply lib/loaders/file_translation_loader.dart:25]: With the new changes, the old get locale method isn't enough.
I'm working right now on the second part of the PR, in order to align this behavior.
BTW, I would like an opinion on what I did on the delegate.

Comment (defint) [reply lib/loaders/file_translation_loader.dart]: Seems like OK.
Will wait the second part, after that we will test it generally. 
Also I can try to create e2e tests with Flutter Drive to test reloading language after changing system locale.

Comment (ilteoood) [reply lib/loaders/file_translation_loader.dart]: I've made the second, and final, part of this PR yesterday.
Ok for Flutter Drive, I would like to use it too, but I suggest to do this kind of work in another PR: we should think also to the CI part

File: lib/loaders/translation_loader.dart [hunk 2]
```diff
@@ -8,11 +8,15 @@ abstract class TranslationLoader {
...
 
-  set locale(Locale locale);
+  set forcedLocale(Locale forcedLocale) => _forcedLocale = forcedLocale;
 
-  Future<Locale> findCurrentLocale() async {
+  Locale get locale => _forcedLocale ?? _locale;
+
+  set locale(Locale locale) => _locale = locale;
+
+  Future<Locale> findDeviceLocale() async {
```
Comment (defint) [inline lib/loaders/translation_loader.dart:19]: In future I would like to rewrite `findSystemLocale` to Flutter's version of locale recognizing.

**Pull request merged**